controllers/dnspolicy_status.go

```diff
@@ -41,37 +34,7 @@ import (
 
 var NegativePolarityConditions []string
```

Comment:
What `NegativePolirity` meats for the record condition?

Reply:
I'll defer that question to @philbrookes who added it.

I'm only removing code from this file in this PR, this already exists.